addons/binding/org.openhab.binding.logreader/README.md

```diff
@@ -0,0 +1,107 @@
+# LogReader Binding
+
+Binding to read and analyze log files. Search patterns are fully configurable, therefore different kind of log files should be possible to monitor by this binding. When certain log events is recognized, openHAB rules can be used to send notification about the event e.g by email for further analyze.
+
+## Supported Things
+
+At this time Binding supports only one ThingType: `reader`. One reader support 3 separated channels; One for errors, one for warnings and one custom channel for other purposes. If you need more channels, you can always add another Thing.
```

Comment:
... this binding supports...

Reply:
... A reader supports 3 separate channels...

Reply:
> If you need more channels, you can always add another Thing.

Hey, why don't you use extensible Things?

Reply:
Done, partly